addons/telestation/fn_customFindSafePos.sqf

```diff
@@ -150,7 +154,9 @@ for "_i" from 1 to MAX_TRIES do
             {
                 private _pos = (AGLtoASL _this) vectorAdd [0, 0, 1.8]; // eye level to eye level check
                 private _relDir = _x getRelDir _pos;
-                if ((_relDir < 60 or _relDir > 300) && {[_x, "GEOM"] checkVisibility [eyePos _x, _pos] > 0.01}) exitWith {true}; false // check unit is facing position and position is visible
+                private _visibility = if !(side _x in [_friendlySide, civilian]) then {[_x, "GEOM"] checkVisibility [eyePos _x, _pos]} else {0};    // It is ok if friendly units can see the TP position, but not enemies.
```

Comment:
Wouldn't it be better to actually check for side friendliness (which would implicitly include civilian too) rather than assuming only own+civilian side is friendly? .. `BIS_fnc_sideIsFriendly` or `side _x getFriend side player >= 0.6` directly.